taxi/urls.py

```diff
@@ -42,7 +47,28 @@
     path("cars/create/", CarCreateView.as_view(), name="car-create"),
     path("cars/<int:pk>/update/", CarUpdateView.as_view(), name="car-update"),
     path("cars/<int:pk>/delete/", CarDeleteView.as_view(), name="car-delete"),
+    path(
+        "cars/<int:pk>/assign/",
+        CarAssignView.as_view(),
+        name="driver-assign"
+    ),
+    path(
+        "cars/<int:pk>/resign/",
+        CarResignView.as_view(),
+        name="driver-resign"
+    ),
     path("drivers/", DriverListView.as_view(), name="driver-list"),
+    path("drivers/create/", DriverCreateView.as_view(), name="driver-create"),
+    path(
+        "drivers/<int:pk>/delete/",
+        DriverDeleteView.as_view(),
+        name="driver-delete"
+    ),
+    path(
+        "drivers/<int:pk>/update-license/",
+        DriverUpdateLicenseView.as_view(),
+        name="driver-update"
+    ),
     path(
         "drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail"
     ),
```

Comment:
Code Style issue: The same issue as above. Change `driver-detail` to `driver_detail`.